src/components/SideMenu/Tab.tsx

```diff
@@ -0,0 +1,19 @@
+import './Tab.styles.scss';
+
+interface TabProps extends HTMLElement {
```

Comment:
`HTMLAnchorElement` then don't need href prop to be declared

Reply:
done